healsparse/healSparseMap.py

```
@@ -262,3 +262,17 @@ def generateHealpixMap(self, nside=None, reduction='mean'):
 
         pass
 
+    def degrade(self, nside_out, reduction='mean'):
+        """
+        Reduce the resolution, i.e., increase the pixel size
+        of a given sparse map
+        """
+        try:
+           assert(self._nsideSparse > nside_out)
+        except ValueError as e:
+           print('nside_out should be smaller than nside for the sparseMap')
```

This should actually raise an exception.  Try:
```
if self._nsideSparse > nside_out:
    raise ValueError('nside_out should be smaller than nside for the sparseMap')
```